mantid-developer-linux.yml

```diff
@@ -14,6 +14,7 @@ dependencies:
   - gsl=2.6 # Keep gsl a specific version to reduce changes in our fitting
   - h5py>=2.10.0,<3 # Pinned back due to api differences
   - hdf5=1.10.*
+  - ipykernel<6 # Added a hard dependency to deal with an issue caused by ipykernel in v6
```

Comment:
Can we clarify the comment to be slightly more descriptive of the issue?

Reply:
Something along the lines of this?
`Added as hard dependency only to deal with an issue with ipykernel v6 and incompatibility with qtconsole/workbench, was previously a soft dependency that was allowed to update to v6 by qtconsole, at present it stops workbench from loading.`

Reply:
Sounds good. Thanks for providing the detail.

Reply:
Done!